### DISCUSSION TITLE
Group suddenly only playing through one device

### DISCUSSION THREAD

**Discussion by JimmyMPage** [music-assistant/support#1588] [Q&A] [answered]
I am using WiiM devices for whole home audio. They are connected to various AVRs and powered speakers. Until this morning, MA has been flawlessly playing an automated playlist in the mornings as we wake up.

Today the power went out, and when it all came back up, I couldn't NOT get the group to play...only one player would play even though MA showed the group playing.

I've removed and reinstalled MA, rebooted all the WiiMs, and cleaned out/re-did my Home Assistant automations. Nothing seems to work. The WiiMs group and play fine from the WiiM app. The logs look clean for MA, no errors ... just "Start Queue Flow Stream" messages for the group in green...

It's gotta be something simple....I've restarted everything multiple times...but I can't find anything obvious.

Thanks for any help the community can provide...

**Comment by JimmyMPage** [comment]
As I've been testing...I found the following:

1. I can link the players in the UI and all of them play fine and in sync.
2. When I create a Universal Group Player for them (which is what I've been using to group them for a few weeks now and they have been working great together), I can't get them to play together. One player will play, the rest show as playing but remain idle.
3. If I power off the UGP, the individual players do not return to the player list in the UI and I have to restart the server in Home Assistant - Add-Ons to get them back.

**Reply by OzGav** [reply]
Anything in he logs?

**Reply by JimmyMPage** [reply]
Yes, I have these Assertion Errors in my log:

```2023-11-03 19:20:02.880 ERROR (MainThread) [asyncio] Task exception was never retrieved
future: <Task finished name='Task-332590' coro=<AlbumsController.add_item_to_library() done, defined at /usr/local/lib/python3.11/site-packages/music_assistant/server/controllers/media/albums.py:91> exception=AssertionError()>
Traceback (most recent call last):
  File "/usr/local/lib/python3.11/site-packages/music_assistant/server/controllers/media/albums.py", line 104, in add_item_to_library
    await self.mass.metadata.get_album_metadata(item)
  File "/usr/local/lib/python3.11/site-packages/music_assistant/server/controllers/metadata.py", line 145, in get_album_metadata
    if metadata := await provider.get_album_metadata(album):
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/music_assistant/server/providers/theaudiodb/__init__.py", line 138, in get_album_metadata
    assert isinstance(artist, Artist)
AssertionError
```

I'm not sure what to do with that...but any help is appreciated.

**Reply by senderbell** [reply]
I have the same problem. Only one player from the group is playing.

**Reply by OzGav** [reply, marked as answer]
Ok I am moving this to an issue. @senderbell do you have the same error in the logs? Please add all comments to the issue now and confirm which speakers you are using
